### PR TITLE
Document GJT endpoints

### PR DESCRIPTION
**Pull request by bikmazefe**
🚧 Do not merge until **March 1st**

Ticket: https://memsource.tpondemand.com/entity/87009-gjt-document-job-templates-and-locale

### CODE REVIEW

File: paths/organization_job_template_locales/create.yaml [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+            example:
+            - abcd1234cdef1234abcd1234cdef1234
+          reviewer_ids:
+            description: Array of reviewer ids to be assigned to the job template locale
+            type: array
+            items:
+              type: string
+            example:
+            - abcd1234cdef1234abcd1234cdef1234
+          translator_team_ids_ids:
```
Comment (strzibny) [inline paths/organization_job_template_locales/create.yaml:76]: This should be just `translator_team_ids` (extra _ids)

File: paths/organization_job_template_locales/update.yaml [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+            example:
+            - abcd1234cdef1234abcd1234cdef1234
+          reviewer_ids:
+            description: Array of reviewer ids to be assigned to the job template locale
+            type: array
+            items:
+              type: string
+            example:
+            - abcd1234cdef1234abcd1234cdef1234
+          translator_team_ids_ids:
```
Comment (strzibny) [inline paths/organization_job_template_locales/update.yaml:78]: Here as well, just `translator_team_ids`

File: schemas/organization_job_template.yaml [hunk 3]
```diff
@@ -0,0 +1,23 @@
+---
+organization_job_template:
```
Comment (strzibny) [inline schemas/organization_job_template.yaml:2]: I think this miss the user and team list no?

**Comment by theSoenke** [comment]
@bikmazefe I've just merged a PR with a couple of additional linting steps. Maybe you can try to rebase this branch on master to check that all checks are passing: https://github.com/phrase/openapi/pull/292

File: paths/organization_job_template_locales/create.yaml (hunk 1 above)
Comment (bikmazefe) [reply paths/organization_job_template_locales/create.yaml]: "Find & replace" fail 🤦 Thanks!

**Review by strzibny — APPROVED** [review]

**Review by strzibny — APPROVED** [review]

**Pull request merged**